Assets/Developers/Lab/Systems/DayNightSystem.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Unity.Burst;
+using Unity.Collections;
+using Unity.Entities;
+using Unity.Jobs;
+using Unity.Mathematics;
+using Unity.Transforms;
+
+namespace JanusVagabond
+{
+
+  [UpdateInGroup(typeof(SimulationSystemGroup))]
+  public class DayNightSystem : JobComponentSystem
+  {
+    BeginInitializationEntityCommandBufferSystem m_EntityCommandBufferSystem;
+    protected override void OnCreate()
+    {
+      // Cache the BeginInitializationEntityCommandBufferSystem in a field, so we don't have to create it every frame
+      m_EntityCommandBufferSystem = World.GetOrCreateSystem<BeginInitializationEntityCommandBufferSystem>();
+    }
+
+    // [BurstCompile]
+    struct SpawnJob : IJobForEachWithEntity<Data.DayNight, LocalToWorld>
+    {
+      // Add fields here that your job needs to do its work.
+      // For example,
+      //    public float deltaTime;
+
+      public EntityCommandBuffer.Concurrent CommandBuffer;
+
+      public void Execute(Entity entity, int index, [ReadOnly] ref Data.DayNight data,
```

Comment:
This system doesn't look like it's doing much. But I think we discussed this during the call, so I expect you're working on it